blog/_posts/2015-12-2-includeos-is-now-open-source.md

```diff
@@ -0,0 +1,38 @@
+#IncludeOS is now free and open source
+
+We've finally opened the lid on [IncludeOS](http://www.includeos.org), just in time for the [IEEE CloudCom](http://2015.cloudcom.org/) paper presentation today. We've done quite a lot of work since the paper was written, so here's an update on what IncludeOS is now, and what you can expect in the near future. 
+
+## It's a bit like a java-vm, but for x86 C++
+A Java Virtual Machine is a portable language runtime environment. Java becomes portable across hardware architectures and operating systems, because it uses a common instruction set. Once you've started a Java program, you can't log into it (unless your program itself provides the facilities), and you can't boot up any other programs inside it.
+
+IncludeOS is like a language runtime for C++ programs, compiled into the common x86 instruction set. This has the obvious advantage of removing one layer of abstraction, compared to Java; with hardware virtualization the code will execute directly on the CPU. Like with a Java program, and other unikernels, an IncludeOS service is a program, with the language runtime attached. It's called IncludeOS because you start by saying `#include <os>` and whatever your service needs from the operating system will be included into the virtual machine. 
+
+Think about the original reasons for creating Java; one main reason was definitely portability. With pervasive hardware virtualization you can now get similar portability for compiled C++ programs. We're currently only portable across x86 operating systems, but with virtualbox, you can run the same binary on windows, Linux and Mac. What's more; "Hello World" in IncludeOS on KVM, will have a much smaller memory footprint than "Hello world" in Java, running directly on the server - even when we take the whole Qemu process into account.
+
+From this interesting premise, rather than working towards running a Java Virtual Machine, or supporting another high-level language, we want to focus on creating an intuitive and modern C++ hardware API, with minimal overhead. But of course - if somebody wants to use IncludeOS as a platform for supporting high level language runtimes, that would be great too. Before that can happen though, we'll have to get a stable API -  we're currently on 0.7.0, meaning that anything can change at any time.
```

Comment:
Java Virtual Machine -> JVM (?) -- could lose the comma after Machine too
course - -> course,
stable API - -> stable API.